### PR TITLE
AirfRANS: gc=1.5 + no weight decay at 4L/256d

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
The current best (0.007264) uses WD=1e-2 with gc=1.0. We want to test whether removing weight decay entirely (WD=0) while using gc=1.5 can further improve results. Weight decay is a form of L2 regularization that shrinks weights toward zero — but with gradient clipping providing implicit regularization, WD may be redundant or even counterproductive for this architecture. With 4L/256d, the model has significant capacity, and removing WD lets the optimizer explore more freely while gc=1.5 prevents gradient explosions.

## Instructions
Train on AirfRANS with 4L/256d architecture, gc=1.5, no weight decay, T_max=5:

```bash
cd target/icml2026 && python train.py \
  --dataset airfrans \
  --num-layers 4 \
  --hidden-dim 256 \
  --use-fourier-features \
  --no-ema \
  --optimizer adamw \
  --lr 7e-4 \
  --weight-decay 0.0 \
  --grad-clip 1.5 \
  --scheduler cosine \
  --T-max 5 \
  --epochs 50 \
  --wandb_group airfrans-gc15-nowd
```

Key changes vs baseline:
- `--grad-clip 1.5` (baseline: 1.0)
- `--weight-decay 0.0` (baseline: 1e-2)
- Everything else identical to the current best

Report `val_primary/surface_mse` at end of training.

## Baseline
Current best metrics (AirfRANS):
- `val_primary/surface_mse`: **0.007264**
- Architecture: 4L/256d + Fourier, no-EMA, AdamW, lr=7e-4, WD=1e-2, gc=1.0, T_max=5
- Baseline PR: #2727 (shoya)
- Reproduce: `cd target/icml2026 && python train.py --dataset airfrans --num-layers 4 --hidden-dim 256 --use-fourier-features --no-ema --optimizer adamw --lr 7e-4 --weight-decay 1e-2 --grad-clip 1.0 --scheduler cosine --T-max 5 --epochs 50`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2785 because ADVISOR: Closing — gc=1.5 is confirmed dead at 4L/256d even without WD (kohaku #2767 confirmed: 0.024 vs 0.007264 baseline)..
